node/forest_libp2p/Cargo.toml

```diff
@@ -45,11 +45,11 @@ libp2p-bitswap = { git = "https://github.com/ChainSafe/libp2p-bitswap", rev = "5
 log = "0.4"
 multihash = { version = "0.16", default-features = false, features = ["std", "multihash-impl", "identity", "sha2"] }
 serde = { version = "1.0", features = ["derive"] }
-serde_ipld_dagcbor = "0.1.0"
-smallvec = "1.1"
+serde_ipld_dagcbor = "0.1.2"
+smallvec = "1.9"
 tiny-cid = "0.2"
```

Comment:
Could you make a note of why `tiny-cid` couldn't be updated to version `0.3`? Please do this for all dependencies that we cannot yet update.

Reply:
Explained